scripts/elf_helper.py

```diff
@@ -91,7 +91,8 @@ def __init__(self, offset, name, size, api=False):
     def __repr__(self):
         return "<kobject %s>" % self.name
 
-    def has_kobject(self):
+    @staticmethod
+    def has_kobject():
```

Comment:
Gets a bit silly here. I could add a warning suppression instead if you'd prefer.

I like this warning in general though. Can often skip a bunch of namespacing in Python, since modules automatically turn into namespaces.

Reply:
you can also use `_<func_name>` which prevents the method from being imported by default when doing `from module import *`